src/en/RoyalRoad.lua

```diff
@@ -1,11 +1,124 @@
--- {"id":36833,"ver":"1.0.5","libVer":"1.0.0","author":"TechnoJo4","dep":["url>=1.0.0","CommonCSS>=1.0.0"]}
+-- {"id":36833,"ver":"1.0.6","libVer":"1.0.0","author":"TechnoJo4","dep":["url>=1.0.0","CommonCSS>=1.0.0"]}
 
 local baseURL = "https://www.royalroad.com"
 local qs = Require("url").querystring
 
 local css = Require("CommonCSS").table
 
+local KEYWORD_FILTER_KEY = 099
+local AUTHOR_FILTER_KEY = 100
 local GENRES_FILTER_EXT = {"Action", "Adventure", "Comedy", "Contemporary", "Drama", "Fantasy", "Historical", "Horror", "Mystery", "Psychological", "Romance", "Satire", "Sci-fi", "Short Story", "Tragedy"}
+local GENRES_FILTER_KEY = 200
+local GENRES_FILTER_INT = { --individual, start from 1 to match ext
+	[201]="action",
+	[202]="adventure",
+	[203]="comedy",
+	[204]="contemporary",
+	[205]="drama",
+	[206]="fantasy",
+	[207]="historical",
+	[208]="horror",
+	[209]="mystery",
+	[210]="psychological",
+	[211]="romance",
+	[212]="satire",
+	[213]="sci_fi",
+	[214]="one_shot",
+	[215]="tragedy",
```

Comment:
```suggestion
	[GENRES_FILTER_KEY+1] = "action",
	"adventure",
	"comedy",
	"contemporary",
	"drama",
	"fantasy",
	"historical",
	"horror",
	"mystery",
	"psychological",
	"romance",
	"satire",
	"sci_fi",
	"one_shot",
	"tragedy",
```

1. When the keys are sequential numbers, you can omit all except the first one.
2. Code style nitpick: put spaces around the `=`.
3. Avoid repeating magic constants.